### DISCUSSION TITLE
Show notification with sound

### DISCUSSION THREAD

**Discussion by rdr6000** [cortezaproject/corteza-server#1838] [Q&A]
Hey I have developed a system using Integration Gateway where a record is added when the endpoint is hit. 

Now I want to notify all the users, that have their browser tab open, "New record added" with sound. 

Any way I can do this?

**Comment by Fajfa** [comment]
Not that i know of currently. Adding sound prompts could be a useful feature. 
Feel free to implement it and contribute to the project, or create a feature request.
